modules/olm-terms.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding_olm/olm-common-terms.adoc
+
+[id="olm-common-terms-bundle_{context}"]
+= Bundle
+In the Bundle Format, a _bundle_ is a collection of an Operator CSV, manifests,
+and metadata. Together they a unique version of an Operator that can be
+installed onto the cluster.
+
+[id="olm-common-terms-bundle-image_{context}"]
+= Bundle image
+In the Bundle Format, a _bundle image_ is a container image built from Operator
```

Comment:
s/image built/image that is built/
s/and contains/and that contains/
Do you need to say "exactly"? Dropping it doesn't seem to change the meaning, so I would.